tests/benchmarks/multicore/idle_uarte/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config GLOBAL_DOMAIN_FREQUENCY_SETTING
+	int "Global Domain frequency, 0 - 320MHz (default), 1 - 256MHz, 2 - 128MHz, 3 - 64MHz"
+	default 0
```

Comment:
this should be a choice, then another (hidden) symbol that has defaults based on what choice symbol is selected

Reply:
This is integer used directly in code to automate the test, choice will make this unnecessarily complicated.

Reply:
How?

C code becomes simpler:
```
const struct nrf_clock_spec clk_specs_global_hsfll = {
		.frequency = MHZ(CONFIG_TEST_CLOCK_SPEED_MHZ),
};
```

Kconfig code:
```
choice TEST_CLOCK_SPEED_OPTION
        prompt "Operational speed"
        default TEST_CLOCK_SPEED_OPTION_320MHZ

config TEST_CLOCK_SPEED_OPTION_320MHZ
        int "320MHz"

config TEST_CLOCK_SPEED_OPTION_256MHZ
        int "256MHz"

config TEST_CLOCK_SPEED_OPTION_128MHZ
        int "128MHz"

config TEST_CLOCK_SPEED_OPTION_64MHZ
        int "64MHz"

endchoice

config TEST_CLOCK_SPEED_MHZ
        int
        default 320 if TEST_CLOCK_SPEED_OPTION_320MHZ
        default 256 if TEST_CLOCK_SPEED_OPTION_256MHZ
        default 128 if TEST_CLOCK_SPEED_OPTION_128MHZ
        default 64 if TEST_CLOCK_SPEED_OPTION_64MHZ
```